client/src/App.js

```diff
@@ -1,39 +1,32 @@
-import logo from './logo.svg';
+import React, { useState, useEffect } from 'react';
 import './App.css';
 
 import axios from 'axios';
 
+import YarntypeList from './components/YarntypeList';
+
 function App() {
-  
-  const callServer = () => {
-    console.log("clicked call server button");
-    axios
-      .get("http://localhost:3000/api")
-      .then(res => {
-        console.log('res', res);
-      })
-      .catch(err => {
-        console.log('err', err)
-      })
-  }
+  const [yarntypes, setYarntypes] = useState();
 
-  const getYarn = () => {
-    console.log("attempting to get yarn")
-    axios
-      .get("http://localhost:3000/api/yarntypes")
-      .then(res => {
-        console.log('res', res);
-      })
-      .catch(err => {
-        console.log('err', err)
-      })
-  }
+  useEffect(() => {
+    console.log('Hi, this is useEffect')
+    const getYarntypes = () => {
+      axios
+        .get("http://localhost:3000/api/yarntypes")
+        .then(res => {
+          setYarntypes(res.data);
+        })
+        .catch(err => {
+          console.log('err', err)
```

Comment:
Something to think about for the future. When this is running in production, what will become of these error messages, i.e., who will see them and how will you know that the errors are occurring?